packages/docs-reanimated/docs/guides/worklets.mdx

```diff
@@ -158,3 +158,5 @@ function App() {
 Worklets can run in other runtimes than the one provided by Reanimated. For example [VisionCamera](https://github.com/mrousavy/react-native-vision-camera) and [LiveMarkdown](https://github.com/Expensify/react-native-live-markdown) create their own worklet runtimes.
 
 You can create your own worklet runtimes with [`createWorkletRuntime`](/docs/threading/createWorkletRuntime) function.
+
+There's no separate UI thread available on the Web platform. Because of that, when Reanimated runs in the browser, worklets are resolved to plain JavaScript functions and the `'worklet';` directive has no effect.
```

Comment:
This isn't exactly true. We still depend on the babel plugin for closure capturing to deduce dependencies for styles etc. When doing some advanced stuff on the web not using `worklet` could actually hurt your code. I know it shouldn't be like that, but what can we do 😿

Reply:
thanks! fixed in [cf8d59b](https://github.com/software-mansion/react-native-reanimated/pull/6737/commits/cf8d59b1919c9c826dfa1849d31b1e56d2a274d4)